### DISCUSSION TITLE
return rtsp stream after fallback

### DISCUSSION THREAD

**Discussion by vlpoliakov28** [bluenviron/mediamtx#511] [Questions]
Hi!
After fallback, if broken stream do born, why fallback not return back?

**Comment by aler9** [comment]
Hello, at the moment the fallback feature doesn't return back since it consists into an RTSP redirect, that is very similar to an HTTP redirect: it tells the client the address of a new stream, and the client connects to it.
Therefore, a "go back" feature may be implemented in the future, but only when the fallback parameter points to an internal, relative URL, since at the moment it can point to an external URL (i.e. another RTSP server), which can't be controlled by the server.

[a comment by an automated account is not shown]
